.circleci/config.yml

```diff
@@ -36,6 +36,46 @@ commands:
           paths:
             - node_modules
 
+  build-win:
+    description: 'Build for Windows.'
+    steps:
+      - run: git config --global core.autocrlf false
+      - checkout
+      - restore_cache:
+          keys:
+            - dependency-cache-win3-{{ checksum "package.json" }}
+            # if cache for exact version of 'package.json' is not present, then load the most recent cache
+            - dependency-cache-win3-
+      - run:
+          name: 'Install node'
+          shell: bash.exe
+          command: |
+            nvm install $NODE_VERSION
+            nvm use $NODE_VERSION
+            nvm install-latest-npm
+      - run:
+          name: 'Install dependent modules'
+          shell: bash.exe
+          command: |
+            npm config set scripts-prepend-node-path true
+            npm install
+            npm install -g sfdx-cli@7.45.1
```

Comment:
You should change this to install the latest version of `sfdx-cli`. We had to default it to `7.45.1` because of a performance issue that has been resolved and we'll be changing that in the next days.